src/app/components/story-feed-item-detail/story-feed-item-detail.component.ts

```diff
@@ -40,4 +40,8 @@ export class StoryFeedItemDetailComponent implements OnInit {
       );
     });
   }
+
+  onDeleted(comment_id) {
```

Comment:
Looks good to me but i think you will need to add onDeleted method to CollapsibleListComponent also along with changes to html its file to completely fix the issue

Reply:
@kapillamba4 What do you mean add onDeleted method and changes to its html? I did first change for just add *ngIf in CollapsibleListComponent.  However, the wrapper in StoryFeedItemDetailComponent will still be there.  Nothing bad, just it was not the best way. So I decided to emit the delete item to StoryFeedItemDetailComponent to completely remove it for parent.  Now sure what you mean.  May you elaborate a little bit?